projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cpq/cpq-configuration.e2e-spec.ts

```diff
@@ -420,100 +419,112 @@ context('CPQ Configuration', () => {
+        line.price
       );
-      configuration.checkValueSelected(
-        RADGRP_PROD,
-        ATTR_CAM_BODY,
-        VAL_CAM_BODY_D850
-      );
-      configuration.selectAttribute(
-        ATTR_CAM_BODY,
-        RADGRP_PROD,
-        VAL_CAM_BODY_EOS80D
-      );
-      configuration.checkValueSelected(
-        RADGRP_PROD,
-        ATTR_CAM_BODY,
-        VAL_CAM_BODY_EOS80D
-      );
-      configuration.selectAttribute(ATTR_CAM_LEN, CHKBOX_PROD, VAL_CAM_LEN_SI);
-      configuration.checkValueSelected(
-        CHKBOX_PROD,
-        ATTR_CAM_LEN,
-        VAL_CAM_LEN_SI
-      );
-      configuration.clickAddToCartBtn();
-
-      configurationOverview.checkConfigOverviewPageDisplayed();
-      configurationOverview.checkGroupHeaderDisplayed(GRP_CAM_MAIN, 0);
-      configurationOverview.checkAttrDisplayed(
-        'Camera Body',
-        'Canon EOS 80D',
-        0
-      );
-
-      configurationOverview.checkGroupHeaderDisplayed(GRP_CAM_ACC, 1);
-      configurationOverview.clickContinueToCartBtnOnOP();
-
-      cart.verifyCartNotEmpty();
-
-      configuration.clickOnProceedToCheckoutBtnInCart();
-      configuration.checkoutB2B();
-      configuration.selectOrderByOrderNumberAlias(POWERTOOLS);
-      configurationOverview.checkGroupHeaderDisplayed(GRP_CAM_MAIN, 0);
-      configurationOverview.checkAttrDisplayed(
-        'Camera Body',
-        'Canon EOS 80D',
-        0
+      configuration.checkBundleItemQuantity(
+        cartEntryIndex,
+        bundleItemIndex,
+        line.quantity
       );
     });
-  });
+
+    configuration.clickOnEditConfigurationLink(cartEntryIndex);
+
+    configuration.checkAttributeDisplayed(ATTR_CAM_BODY, RADGRP_PROD);
+    configuration.selectAttribute(
+      ATTR_CAM_BODY,
+      RADGRP_PROD,
+      VAL_CAM_BODY_D850
+    );
+    configuration.checkValueSelected(
+      RADGRP_PROD,
+      ATTR_CAM_BODY,
+      VAL_CAM_BODY_D850
+    );
+    configuration.selectAttribute(
+      ATTR_CAM_BODY,
+      RADGRP_PROD,
+      VAL_CAM_BODY_EOS80D
+    );
+    configuration.checkValueSelected(
+      RADGRP_PROD,
+      ATTR_CAM_BODY,
+      VAL_CAM_BODY_EOS80D
+    );
+    configuration.selectAttribute(ATTR_CAM_LEN, CHKBOX_PROD, VAL_CAM_LEN_SI);
+    configuration.checkValueSelected(CHKBOX_PROD, ATTR_CAM_LEN, VAL_CAM_LEN_SI);
+    configuration.clickAddToCartBtn();
+
+    configurationOverview.checkConfigOverviewPageDisplayed();
+    configurationOverview.checkGroupHeaderDisplayed(GRP_CAM_MAIN, 0);
+    configurationOverview.checkAttrDisplayed('Camera Body', 'Canon EOS 80D', 0);
+
+    configurationOverview.checkGroupHeaderDisplayed(GRP_CAM_ACC, 1);
+    configurationOverview.clickContinueToCartBtnOnOP();
+
+    configuration.checkAmountOfBundleItems(cartEntryIndex, 4);
+    configuration.verifyCartCount(numberOfCartItems);
+  }
+
+  function checkout() {
+    configuration.clickOnProceedToCheckoutBtnInCart();
+    configuration.checkoutB2B();
+  }
+
+  function orderHistory() {
```

Comment:
The same here: why we rap these methods. Why couldn't we call directly them?

Reply:
done